lang/python/python-ble2mqtt/Makefile

```
@@ -0,0 +1,43 @@
+#
+# Copyright (C) 2023
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-ble2mqtt
+PKG_VERSION:=0.1.5
+PKG_RELEASE:=1
+
+PYPI_NAME:=ble2mqtt
+PKG_HASH:=0015cae0c36badb48cbd4a1c8b1a8029e45ab0891a95363fe00624c2629b4510
+
+PKG_MAINTAINER:=Quintin Hill <stuff@quintin.me.uk>
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=LICENSE
+
+include ../pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-ble2mqtt
+  SECTION:=lang
+  CATEGORY:=Languages
+  SUBMENU:=Python
+  TITLE:=Bluetooth to MQTT bridge
+  URL:=https://github.com/devbis/ble2mqtt/
+  DEPENDS:=+python3 \
+  	+python-bleak
+  	+python-aio-mqtt-mod
+  VARIANT:=python3
```

```suggestion
  DEPENDS:=+python3 +python-bleak +python-aio-mqtt-mod
```

the dependency list is short enough, no need to wrap new lines.
`VARIANT:=python3` has been deprecated in py3 makefile.